iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -223,7 +223,10 @@ public WorkdirInstruction WORKDIR() {
     return b.<WorkdirInstruction>nonterminal(DockerLexicalGrammar.WORKDIR).is(
       f.workdir(
         b.token(DockerKeyword.WORKDIR),
-        ARGUMENTS()
+        b.oneOrMore(f.tuple(
+          b.token(DockerLexicalGrammar.WHITESPACE),
+          ARGUMENT()
+        ))
```

Comment:
I can imagen having an `ARGUMENTS()` rule in the end which creates this list of tuples. For now, it's fine like this.

Reply:
👍 For the first/two element I did it like this, but I will definitely create an `ARGUMENTS()` to handle the next ones (also will need to rename to old crap `ARGUMENTS()` :D )